Gaijinizer patch/Map042.txt

```diff
@@ -290,265 +317,293 @@ What!?#afrg
 
  とくべつな　エンプーサ　創られた。
 > CONTEXT: Map042/events/5/pages/1/95/Dialogue/enemy2
-Certainly for Me "Shelter" I and another one
-A special kind of empuse was created.#MTLed
+Certainly... there were... two Empuses...
+specially made... to attack... the shelter...#Ron
 > END STRING
 
 > BEGIN STRING
 さいしょ　わたし。
 
 つぎ　もっと　するどく　こわい　エンプーサ。
 > CONTEXT: Map042/events/5/pages/1/99/Dialogue/enemy2
-I am. Next more and more fearful Empuse.#MTLed
+First... me...
+
+Then... a more... scary version... of me...#Ron
 > END STRING
 
 > BEGIN STRING
 でも　たしか　むこうも　メス……
 
 オス　いなかった　はず……
 > CONTEXT: Map042/events/5/pages/1/103/Dialogue/enemy2
-But I am looking forward to it ...
-There should have been no male ......#MTLed
+But... I'm certain... there never... used to be...
+a male version...#Ron
```

Comment:
I feel like we should replace "used to be" with "was" here because the tenses are odd.... what do you think?